documentation/docs/index.md

```diff
@@ -17,9 +18,9 @@ Vespucci has been tested and is known to work on a wide range of devices, from s
 
 #### Obtaining Vespucci
 
- * From [Google's Play Store](https://play.google.com/store/apps/details?id=de.blau.android) and from [Amazon](http://www.amazon.com/Simon-Poole-vespucci/dp/B00P71C3AW/ref=sr_1_1?s=mobile-apps&ie=UTF8&qid=1447617332&sr=1-1&keywords=vespucci)
+ * From [F-Droid](https://f-droid.org/app/de.blau.android), [Google's Play](https://play.google.com/store/apps/details?id=de.blau.android) and [Amazon](https://www.amazon.com/Simon-Poole-vespucci/dp/B00P71C3AW)
```

Comment:
Please no F-Droid references, we do not have any influence or control over this build.

Reply:
F-Droid is no different than any other 3rd party build. Besides that it also doesn't include a number of useful imagery sources and services (for example Mapillary).